### PR TITLE
Added doc command available

### PR DESCRIPTION
**Pull request by julianosaless**
Added table about command available

### CODE REVIEW

**Comment by NullVoxPopuli** [comment]
:+1:

File: docs/general/getting_started.md [hunk 1]
```diff
@@ -15,6 +15,12 @@ And then execute:
 ```
 $ bundle
 ```
+### Available Commands
+
+|command                               |Description                                                 |
+|--------------------------------------|------------------------------------------------------------|
+| rails g resource + `your model`      | Create a new serializer is to generate a new resource.     |
```
Comment (beauby) [inline docs/general/getting_started.md]: This should probably be a note that, by default, a serializer will be generated for every new resource.

File: docs/general/getting_started.md [hunk 2]
```diff
@@ -15,6 +15,12 @@ And then execute:
 ```
 $ bundle
 ```
+### Available Commands
```
Comment (beauby) [inline docs/general/getting_started.md]: I like that we add some rails-specific notes, but I thing they should rather be at the end of the file than at the top.

File: docs/general/getting_started.md (hunk 1 above)
Comment (julianosaless) [inline docs/general/getting_started.md]: @beauby  i totally agree with you, i will change  this line;

File: docs/general/getting_started.md (hunk 2 above)
Comment (julianosaless) [inline docs/general/getting_started.md]: @beauby  What you think about this point  below?

when new user are reading the documentation , I think him needs is all the commands available after the that him  can do with these command.

Comment (beauby) [inline docs/general/getting_started.md]: Thing is, not all new users are rails users. Plus, most rails developers rarely use generators. I don't have a definitive opinion on this though, let's ask @joaomdmoura.

Comment (julianosaless) [inline docs/general/getting_started.md]: @joaomdmoura  any position about  my pr :)

**Comment by beauby** [comment]
Actually @julianosaless, after looking a bit more into this PR, it seems the information you added is already present in the next paragraph: https://github.com/julianosaless/active_model_serializers/blob/add-doc-available/docs/general/getting_started.md#creating-a-serializer

**Comment by bf4** [comment]
Closing as it appears to now be addressed by the docs in `docs/general/getting_started.md` and `lib/generators/rails/USAGE`. Sorry for lost time @julianosaless

**Pull request closed without merging**